config/schema.yaml

```diff
@@ -1,6 +1,6 @@
 # Stage variables
 VERBOSE_COMMANDS:
-  description: Adjust the output verbosity level of Magento commands.
+  description: "Adjust the output verbosity level of Magento commands."
```

Comment:
There is no need in quotes - https://yaml.org/spec/1.2/spec.html#id2760844

Reply:
http://blogs.perl.org/users/tinita/2018/03/strings-in-yaml---to-quote-or-not-to-quote.html

Reply:
Some description messages contain colon symbol `:` and without quotes, `:<space>` won't work.
I just changed all description messages to one style.

Reply:
This will be lost on the dump in any case, please remove colons where they’re not needed

Reply:
Colon won't be lost on the dump. We have a lot of variables with such construction `_merge: true` in the description. Also, there is one message which requires newlines, without quotes it won't work too.

Reply:
Quotes will be lost, not colons. Yaml Dump class by default follows yaml spec and won’t dump quotes

Reply:
Yes, I understand that quotes will be lost. But as I mentioned some descriptions won't work without quotes, and I just changed all descriptions to one format.